### PR TITLE
0.14.3 release notes, manpage and version bump

### PR DESCRIPTION
**Pull request by Sjors**
I left a bunch of test and documentation commits out of the release notes.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -103,49 +46,68 @@ smart fee slider. It will only present an approximate fee calculated using the
...
+
+- #9481 `7abe7bb` Give fallback fee a reasonable indent (Luke Dashjr)
+- #9481 `3e4d7bf` Qt/Send: Figure a decent warning colour from theme (Luke Dashjr)
+- #9481 `e207342` Show more significant warning if we fall back to the default fee (Jonas Schnelli)
 
 ### Wallet
-- #10294 `1847642` Unset change position when there is no change (instagibbs)
+
+ - #10308 `28b8b8b` Securely erase potentially sensitive keys/values (tjps)
+ - #10265 `ff13f59` Make sure pindex is non-null before possibly referencing in LogPrintf call. (Karl-Johan Alm)
```
Comment (ken2812221) [inline doc/release-notes.md]: Some PR has space in front of `-`. Is that intentional?

Comment (Sjors) [reply doc/release-notes.md]: Fixed.

**Comment by laanwj** [comment]
utACK 02857b121c13b6ffebcf4c83771b34c1124e7bd2

**Comment by ken2812221** [comment]
utACK 02857b121c13b6ffebcf4c83771b34c1124e7bd2

**Pull request merged**